File: server/src/main/kotlin/org/kmm/sample/model/ExpenseModel.kt
package org.kmm.sample.model

import kotlinx.serialization.Serializable

/**
 * Created by Charles Raj I on 02/01/25
 * @project KmmSample
 * @author Charles Raj
 */


@Serializable
class ExpenseModel (
     val id : Int? = null,
    val amount: Double? = null,
    val type: String? = null,
    val description : String? = null
)

File: iosApp/iosApp/ExpenseModel.swift
//
//  ExpenseModel.swift
//  iosApp
//
//  Created by charles raj on 05/01/25.
//  Copyright © 2025 orgName. All rights reserved.
//

import Foundation

class ExpenseModel: Codable {
    let id: Int
    let amount: Double
    let type: Date
    let description: String
}
